### DISCUSSION TITLE
How to upload .csv with boolean values?

### DISCUSSION THREAD

**Discussion by RawmanGrey** [nocodb/nocodb#1832] [Q&A]
I've tried setting boolean values as `true, "true", True, "True", 1, "1"`, but with no success facing "Source data contains some invalid boolean values" error again and again. If I export data and import it again all boolean values interpreted correctly, but saving any changes with my spreadsheets or text editors brings me back to wrong value error. 

Resolved #880 makes me think I'm doing wrong with file settings. Ensuring on Unicode UTF-8 encoding and setting same as in export Windows-like line endings was not enough. What else has to be done?

File I'm failing to upload - [simple.csv](https://github.com/nocodb/nocodb/files/8540827/simple.csv)

Please, guide me to proper .csv file settings : )

**Comment by o1lab** [comment]
Thank you for the details. 

Upload csv is in alpha. We will be improving it :)
